soundcard/mediafoundation.py

```diff
@@ -15,7 +15,10 @@
 _combase = _ffi.dlopen('combase')
 _ole32 = _ffi.dlopen('ole32')
 
+
+
 class _COMLibrary:
+    comLoadedByThisClass = False;
```

Comment:
Please rename to a pythonic name, i.e. `external_com` or some such.

Reply:
Renamed in new PR